src/templates/erc20/erc20.ts

```diff
@@ -69,7 +74,8 @@ export const createERC20Tracker = ({
           contract.decimals(),
         ])
         erc20 = new ERC20({
-          id: address,
+          id: `${ctx.chain.id}-${address}`,
```

Comment:
what do you think of adopting the same id format than in FE:
``${symbol}${chainId === mainnet.id ? '' : `:${chainId}`}``
Maybe wrap this in `getTokenId/parseTokenId` utils

Reply:
For using symbol, I'm unsure - address feels more official, yet we should only ever be officially recognizing specific symbols so it could be OK.

Regarding placement of the chainId, I would rather have it in front because it's the larger scale item.

Synchronizing these would be nice though I agree.

Reply:
yeah, I'm not against switching place for chain and address, this is how gnosis safe is doing as well... For the symbol vs address point, it's just to optimize readability, it's easier with the symbol obviously